src/24-uncensorable-wikipedia/index.md

```diff
@@ -0,0 +1,199 @@
+* Turkish Wikipedia (most recent snapshot, resolved through IPNS): [/ipns/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S/wiki/Anasayfa.html](https://ipfs.io/ipns/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S/wiki/Anasayfa.html)
+
+### Human-readable links
+
+For your convenience we have set up a domain name and DNS entry at [tr.wikipedia-on-ipfs.org](http://tr.wikipedia-on-ipfs.org) that will resolve to the current IPFS snapshot.
+
+If you are not able to access ipfs.io or wikipedia-on-ipfs.org, you can use this dnslink URL to access the content -- simply replace `ipfs.io` with the address of any IPFS gateway:
+
+- Turkish Wikipedia: [https://ipfs.io/ipns/tr.wikipedia-on-ipfs.org](https://ipfs.io/ipns/tr.wikipedia-on-ipfs.org/Anasayfa.html)
+
+Failing that, use the (less readable) IPFS or IPNS links from above.
+
+
+## How to access Wikipedia on IPFS
+
+IPFS makes it possible to access information through many different paths. If one path is closed, you have other options. Some of the options are very simple.  Others require a bit more effort. In most cases you can use [this link](https://ipfs.io/ipfs/Qme2sLfe9ZMdiuWsEtajWMDzx6B7VbjzpSC2VWhtB6GoB1/wiki/Anasayfa.html) to view the Wikipedia snapshot through the ipfs-to-http gateway at ipfs.io. If that path doesn't work, for example if ipfs.io is blocked in your area, you can use the same basic information from that ipfs.io link and access Wikipedia through a different path.
+
+We've written a number of tutorials describing the many different ways to access content through IPFS, including ways to access IPFS anonymously through Tor. You can read those tutorials in the current draft of [The Decentralized Web Primer](https://dweb-primer.ipfs.io/avenues-for-access/), which is available [online](https://dweb-primer.ipfs.io/avenues-for-access/) or as a [downloadable PDF](https://dweb-primer.ipfs.io/decentralized-web-primer.pdf), [epub](https://dweb-primer.ipfs.io/decentralized-web-primer.epub), or [mobi](https://dweb-primer.ipfs.io/decentralized-web-primer.mobi)
+
+Your main options for accessing the snapshot of Wikipedia are: _(depending on your network, some of these won't work)_
+
+* **Option**: Use the ipfs.io gateway to access the 30 April 2017 snapshot: https://ipfs.io/ipfs/Qme2sLfe9ZMdiuWsEtajWMDzx6B7VbjzpSC2VWhtB6GoB1/wiki/Anasayfa.html
+* **Option**: Use the ipfs.io gateway to access the latest version: https://ipfs.io/ipns/QmQP99yW82xNKPxXLroxj1rMYMGF6Grwjj2o4svsdmGh7S/wiki/Anasayfa.html
+* **Option**: Connect with the IPFS network over Tor (this is experimental. Read [this tutorial on Tor gateways](https://dweb-primer.ipfs.io/avenues-for-access/lessons/tor-gateways.html) or [this tutorial on running IPFS over Tor transport](https://dweb-primer.ipfs.io/avenues-for-access/lessons/tor-transport.html))
```

Comment:
Is there still room to change this? "this is experimental" is not a correct statement, since Onion support in go-ipfs does not exist. It does exist in a fork of go-ipfs (which is OpenBazaar).

Sorry I'm late here, I forgot about filing it the other day.